install_config/redeploying_certificates.adoc

```diff
@@ -413,6 +413,11 @@ $ ansible-playbook -i <inventory_file> \
     playbooks/openshift-master/redeploy-certificates.yml
 ----
 
+[NOTE]
+====
+After running this playbook, you must update the xref:../install_config/certificate_customization.adoc#configuring-custom-certificates[named certificate parameters] in the *_master-config.yaml_* file on each master node. Then, restart the OpenShift Container Platform master services to apply the changes.
```

Comment:
Can you replace "OpenShift Container Platform" with "{product-title}"? (That way it will show either "OpenShift Container Platform" or "OKD", etc. depending on which doc set it is).